apps/consent-ui/src/app/consent-api/[...proxy]/route.ts

```diff
@@ -0,0 +1,114 @@
+/*
+ * Copyright (c) 2024 The Ontario Institute for Cancer Research. All rights reserved
+ *
+ * This program and the accompanying materials are made available under the terms of
+ * the GNU Affero General Public License v3.0. You should have received a copy of the
+ * GNU Affero General Public License along with this program.
+ *  If not, see <http://www.gnu.org/licenses/>.
+ *
+ * THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS" AND ANY
+ * EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE IMPLIED WARRANTIES
+ * OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE ARE DISCLAIMED. IN NO EVENT
+ * SHALL THE COPYRIGHT HOLDER OR CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT,
+ * INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED
+ * TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR PROFITS;
+ * OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER
+ * IN CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN
+ * ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+ */
+
+import axios, { AxiosError, AxiosHeaders } from 'axios';
+import { NextRequest, NextResponse } from 'next/server';
+import { auth } from 'src/app/auth';
+import { getAppConfig } from 'src/config';
+import { decryptContent } from 'src/services/api/utils';
+
+import urlJoin from 'url-join';
```

Comment:
Smol linter change to organize the imports:

```suggestion
import axios, { AxiosError, AxiosHeaders } from 'axios';
import { NextRequest, NextResponse } from 'next/server';
import urlJoin from 'url-join';

import { auth } from 'src/app/auth';
import { getAppConfig } from 'src/config';
import { decryptContent } from 'src/services/api/utils';
```

Reply:
👍 
fixed in another commit.